media/js/firefox/australis/tour.js

```diff
@@ -3,6 +3,12 @@
 
     var firstTime = 'True';
 
+    // GA tracking for Firstrun tests
```

Comment:
Oh, this is for the default test case, right? In that case, just adding a version check for 35.0.1 should be enough